react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicked: 0, color: 'white' };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    const newClickNum = this.state.clicked + 1;
+
+    this.setState({ clicked: newClickNum });
+
+    const colorList = ['darkblue', 'purple', 'hotpink', 'orange', 'yellow', 'white'];
+    let newColor = '';
+    if (Number.isInteger((newClickNum) / 3) && (newClickNum) < 19) {
+      newColor = colorList[(newClickNum) / 3 - 1];
+      this.setState({ color: newColor });
+    }
+  }
+
+  render() {
+    return (
```

Comment:
Here you can call your newly created `getClass` method to get the current color class. It would look something like:

```JSX
const colorClass = this.getClass();
```

You can then use the `colorClass` constant to set the `className` on your button.